hax/__init__.py

```diff
@@ -1,11 +1,12 @@
 import logging
 import os
+import ast
```

Comment:
This is now unused, though I suppose you could replace the eval of the main config with this (currently it has a "slightly" less secure eval with the `os` module exposed (which is also no longer used))

Reply:
Just removed in 6ad82f6 (since I couldn't easily just swap ```eval``` with ```ast.literal_eval```).